src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -191,20 +193,47 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
       }
     }
 
-    if (this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers) {
+    const isSimulcastPolicy = this.isSimulcastUplinkPolicy(
+      this._configuration.videoUplinkBandwidthPolicy
+    );
+    let enableSimulcast =
+      this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers || isSimulcastPolicy;
+
+    if (enableSimulcast) {
       if (!this._configuration.enableUnifiedPlanForChromiumBasedBrowsers) {
-        this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
         this.logger.info(
           'Simulcast requires enabling WebRTC Unified Plan for Chromium-based browsers'
         );
-      } else if (browserBehavior.hasChromiumWebRTC()) {
+        enableSimulcast = false;
+      } else if (this._configuration.videoUplinkBandwidthPolicy && !isSimulcastPolicy) {
+        this.logger.info(
+          'Simulcast requires video uplink bandwidth policy specified in configuration to implement' +
+            ' SimulcastUplinkPolicy interface.'
+        );
+        enableSimulcast = false;
+      }
+      if (browserBehavior.hasChromiumWebRTC()) {
         this.logger.info(`Simulcast is enabled for ${browserBehavior.name()}`);
       } else {
-        this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
         this.logger.info(
           'Simulcast requires WebRTC Unified Plan and is only supported on Chromium-based browsers'
         );
+        enableSimulcast = false;
+      }
+
+      if (!enableSimulcast) {
+        //Clean up if simulcast can't be enabled
+        this._configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
+        if (isSimulcastPolicy) {
+          this._configuration.videoUplinkBandwidthPolicy = undefined;
```

Comment:
Can we add to the logs above to call out that we are removing the set policy?  I'm guessing builders may not look at code and realize what is happening.

Reply:
Opt to throw error now and no longer remove policy :)